test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -599,7 +569,7 @@ public async Task HttpTrigger_DuplicateQueryParams_Succeeds()
                 Assert.Equal(HttpStatusCode.OK, response.StatusCode);
                 string body = await response.Content.ReadAsStringAsync();
                 Assert.Equal("text/plain", response.Content.Headers.ContentType.MediaType);
-                Assert.Equal("Hello, Mathew,Amy", body);
+                Assert.Equal("Hello Mathew,Amy", body);
```

Comment:
nit: is this an intentional change?

Reply:
yes , this is an intentional change as I refactored invoking http trigger across all languages